substrate/pkg/controller/substrate/cluster/addons/fluentbitcw.go

```diff
@@ -288,6 +313,9 @@ spec:
         - name: dmesg
           mountPath: /var/log/dmesg
           readOnly: true
+        - name: audit-log
+          mountPath: /var/log/kubernetes/audit/
+          readOnly: false
```

Comment:
We already mount `/var/log` on line 302, do we need to mount audit directory again?

Reply:
The audit log volume cannot be `readOnly: true`. Did we want to update the `/var/log` volume mount to be `readOnly: false`?

Reply:
Why does Fluentbit needs to write to audit logs file? It should be similar to any other logs file which FluentBit is parsing and pushing to CW

Reply:
Removed the extra mount and tested. Still populating `audit.log` as expected.